src/main/java/frc/robot/RobotContainer.java

```diff
@@ -7,16 +7,20 @@
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Commands;
 import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import frc.robot.pneumatics.Pneumatics;
 
 public class RobotContainer {
-  private final CommandXboxController kPilotController =
-      new CommandXboxController(Constants.kPilotControllerPort);
+  private final CommandXboxController controller = new CommandXboxController(Constants.kPilotControllerPort);
```

Comment:
We can standardize naming of the controller from the CommonBase https://github.com/Prosper-FRC/template-talon_standard/pull/5

Reply:
Okay